groups.py

```diff
@@ -1,29 +1,32 @@
 import asyncio
 import json
 import os
-from spond import spond
-from config import username, password
-
 
-if not os.path.exists('./exports'):
-    os.makedirs('./exports')
+from config import password, username
+from spond import spond
 
+if not os.path.exists("./exports"):
+    os.makedirs("./exports")
 
 
 async def main():
     s = spond.Spond(username=username, password=password)
     groups = await s.get_groups()
     for group in groups:
-        name = group['name']
-        data = (json.dumps(group, indent=4, sort_keys=True))
-        keepcharacters = (' ','.','_')
-        filename = os.path.join("./exports", "".join(c for c in name if c.isalnum() or c in keepcharacters).rstrip() + ".json")
+        name = group["name"]
+        data = json.dumps(group, indent=4, sort_keys=True)
+        keepcharacters = (" ", ".", "_")
+        filename = os.path.join(
+            "./exports",
+            "".join(c for c in name if c.isalnum() or c in keepcharacters).rstrip()
+            + ".json",
+        )
         print(filename)
-        with open(filename, 'w') as out_file:
+        with open(filename, "w") as out_file:
             out_file.write(data)
-        
+
     await s.clientsession.close()
 
+
 loop = asyncio.get_event_loop()
```

Comment:
We shold probably change these to the "new" way to avoid warnings and errors in python 3.10 and 3.11

 ```
loop = asyncio.new_event_loop()
asyncio.set_event_loop(loop)
asyncio.run(main())
```

I believe this works fine in 3.7 as well?

Reply:
This code isn't chnaged by this pull request, so have raised a new issue #46